README.md

```diff
@@ -2,31 +2,31 @@
+The project has been cleaned up, all **prototype elements** that no longer work such as boot, GDT, low memory management ...etc, have been removed from the project! Our apologies if there are any bits or comments mentionning them in the code or any copyrighted parts, you can ignore them. This version runs on top of FreeDOS. This will be adjusted on the next major version (OS2.2)
 
 Platform  | Build Status
  --- | --- 
 Windows | [![Build status](https://ci.appveyor.com/api/projects/status/nfkdxmis9s2s4d4s?svg=true)](https://ci.appveyor.com/project/Maeiky/cpcdosos2-1)
 
 ## How to contribute
 - Create a fork of the repository on your GitHub account.
-- Make your contribution while testing if it works (see the paragraph on compilation below).
-- Make a push of your changes to your fork.
-- Create a pull-request via your fork. AppVeyor will test to see if everything is fine. The repository owner will check and, if all is well, merge your contribution as soon as possible!
+- Make your contribution. Don't forget to test your code before commiting (see the paragraph on compilation below).
+- Push your changes to your fork.
+- Create a pull-request. AppVeyor will make sure that everything is fine. The repository owners will check if everything works as expected, if so, they'll merge your contribution as soon as possible!
 
 ## How to compile
  - Download and install the latest version of [CWC](https://github.com/VLiance/Cwc)
- - Then simply run the `OS2.1\_Make_CpcDos_.cwMake` _associated with CWC_ file
- - Make sure you have a [editable] virtual machine (https://github.com/VLiance/VW_Viewer) or a bootable USB Cpcdos device (Cpcdos Tutorial [No1](https://www.youtube.com/user/cpcdososx) [No2](https://www.youtube.com/channel/UCkFCPxJF7ZzmWxW4i5WavCA/videos))
- It is recommended to use [VMware workstation player 16](https://my.vmware.com/fr/web/vmware/downloads/details?downloadGroup=PLAYER-1610&productId=1039&rPId=55794) and not to have the pro version installed on your machine, as you may have less options to configure your network card.
-   Feel free to use an old cpcdos VM already configured to update it.
+ - Then simply run `OS2.1\_Make_CpcDos_.cwMake` _associated with CWC_ file
+ - Make sure you have an [editable] virtual machine (https://github.com/VLiance/VW_Viewer) or a bootable drive with Cpcdos installed (Cpcdos Tutorial [No1](https://www.youtube.com/user/cpcdososx) [No2](https://www.youtube.com/channel/UCkFCPxJF7ZzmWxW4i5WavCA/videos))
+ It is recommended to use [VMware workstation player 16](https://my.vmware.com/fr/web/vmware/downloads/details?downloadGroup=PLAYER-1610&productId=1039&rPId=55794) make sure that you don't have to pro version on your machine, as you may have less options to configure your network card.
```

Comment:
I think `to` (the ?) is a typo in `make sure that you don't have to pro version`

Reply:
Yep sorry 😅

Reply:
Yep sorry 😅